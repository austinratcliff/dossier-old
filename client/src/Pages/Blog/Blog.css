.Blog {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  background-color: white;
  color: #111;
  font-family: 'IBM Plex Mono', monospace;
  height: 100vh;
  width: 100vw;
}

.Blog .header {
  font-size: calc(10px + 5vmin);
  margin: 0;
}

.Blog .home {
  position: absolute;
  right: 0;
  margin-right: 25px;
  cursor: pointer;
}
