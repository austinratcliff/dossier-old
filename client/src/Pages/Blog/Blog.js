import React from 'react'
import './Blog.css'

const Blog = () => {
  return (
    <div className="Blog">
      <h1 className="header">
        <i
          className="fas fa-align-left"
          title="blog"
        /> Blog
      </h1>
      <i
        className="fas fa-angle-left fa-2x home"
        title="home"
      />
    </div>
  )
}

export default Blog
