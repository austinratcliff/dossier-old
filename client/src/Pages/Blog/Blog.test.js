import React from 'react'
import ReactDOM from 'react-dom'
import ReactTestRenderer from 'react-test-renderer'
import Blog from './Blog'

describe('Blog component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Blog />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('matches snapshot', () => {
    const component = ReactTestRenderer.create(<Blog />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
