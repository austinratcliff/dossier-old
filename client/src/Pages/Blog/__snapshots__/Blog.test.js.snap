// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog component matches snapshot 1`] = `
<div
  className="Blog"
>
  <h1
    className="header"
  >
    <i
      className="fas fa-align-left"
      title="blog"
    />
     Blog
  </h1>
  <i
    className="fas fa-angle-left fa-2x home"
    title="home"
  />
</div>
`;
